src/core/components/choice-card/styles.ts

```diff
@@ -135,10 +145,6 @@ export const choiceCard = ({
 
 	${from.mobileLandscape} {
 		margin: 0 ${space[2]}px 0 0;
-
-		&:last-child {
-			margin: 0;
-		}
```

Comment:
We shouldn't need a margin if we use [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap). Perhaps we can eliminate `margin` altogether when the gridContainer is applied

Reply:
That's kind of tricky, I think.
Margin is associated with the ChoiceCard and that component does not know if it's container is flex or grid (and I don't think it should know). 
If I remove the margin then we can't use flex, unless we use something like `justify-content: space-between` (losing the correct spacing of 8px). 
Another solution is setting the default value of columns to 1 and always use gap.

What do you think?

Reply:
> Margin is associated with the ChoiceCard and that component does not know if it's container is flex or grid (and I don't think it should know).

Agreed. Can we preserve flex as the default and override the margin in `gridContainer`?

```tsx
export const gridContainer = css`
  // ...

  & > label {
    margin: 0;
  }
`
```